docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>You can't be explicit with <code>shape</code>. If you try this:</p><p><code>pm.MvNormal.dist(mu=np.zeros(5), cov=np.eye(5), shape=(3,)).eval()</code> you get back an array length 5. It doesn't error out because shape is incompatible with the implied support. The last entry of shape is effectively ignored. This is the case even when <code>mu</code> has shape <code>(1,)</code> :</p><p><code>pm.MvNormal.dist(mu=[0], cov=[[1.]], shape=(3,)).eval().shape</code> return <code>(1,)</code> not <code>(3,)</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Added a warning about this</p>